docs/reference/ml/trained-models/apis/get-trained-models-stats.asciidoc

```diff
@@ -127,14 +135,31 @@ The deployment stats for each node that currently has the model allocated.
 (double)
 The average time for each inference call to complete on this node.
 
+`error_count`:::
+(integer)
+The number of errors when evaluating the trained model.
+
+
 `inference_count`:::
 (integer)
 The total number of inference calls made against this node for this model.
 
+`inference_threads`:::
+(integer)
+The number of threads used by the inference process.
+This figure may be lower than top level value if there is not sufficient
```

Comment:
> lower than top level value... to satisfy the configured value

It is a bit unclear where you'd find the "top level"/ "configured" value. Is that the value defined in the "inference_threads" option of the start trained model deployment API (for NLP models)? Where is it configured for non-NLP models?

Reply:
`inference_threads` is not a configuration option for non-NLP models, what we see here is inside the `deployment_stats` object which is only returned for NLP models. 

The "top level" value is the one set when [starting](https://www.elastic.co/guide/en/elasticsearch/reference/8.0/start-trained-model-deployment.html#start-trained-model-deployment-query-params) a deployment. Individual nodes are requested to use this many threads but if the hardware does not support that many threads the node will use a lower value which is reported back here. So the "top level" is the number the deployment was started with and the value here is the node's best effort to reach that number. Often they will be the same.

I've had another go at explaining it

Reply:
Thanks!  I've added a few more suggestions. Hope they're useful!